### PR TITLE
Fix 8609 add test

### PR DESCRIPTION
**Pull request by worming004**
All 777 tests are running successfully

### CODE REVIEW

**Comment by worming004** [comment]
Forgot to mention, there was a FilterService required to accept values as readonly to accept the new changes

**Comment by papb** [comment]
Cool!! Thank you very much

File: src/app/components/splitbutton/splitbutton.spec.ts [hunk 1]
```diff
@@ -21,8 +22,16 @@ describe('SplitButton', () => {
...
       splitbutton = fixture.componentInstance;
     });
+
+    it('should accept readonly options', () => {
+        const models : MenuItem[] = [
+            {label:'home', url:'https://www.primefaces.org/primeng/'},
+            {label:'split-button', url:'https://www.primefaces.org/primeng/#/splitbutton'}
+        ];
+        splitbutton.model = models;
+    });
```
Comment (papb) [inline src/app/components/splitbutton/splitbutton.spec.ts]: Did you forget `as const` here?

Comment (worming004) [reply src/app/components/splitbutton/splitbutton.spec.ts]: Yes, it's an error from me. Good catch :)

I'll be able to correct it in few hours

Comment (worming004) [reply src/app/components/splitbutton/splitbutton.spec.ts]: Resolved with 3ebce6a

**Comment by worming004** [comment]
@papb , do not forget to merge this PR in order to have a complete code change to propose to primeng team

**Pull request merged**